app/views/staff/events/configuration.html.haml

```diff
@@ -1,3 +1,6 @@
+- if params[:controller].include?("staff") && user_signed_in?
```

Comment:
By the time these staff templates render, you won't have to check for "staff" in the path or whether the user is signed in. Just rendering the partial should work.